src/components/about/Landing.jsx

```diff
@@ -1,7 +1,30 @@
 import React from "react";
-
+import Image from "next/image";
+import chart from "../../../public/assets/chart.svg";
 const Landing = () => {
-  return <div>Landing</div>;
+  return (
+    <div className="w-full relative top-40 flex h-[50vh] md:h-screen">
```

Comment:
I don't think that relative is needed anywhere in this code. Using it means that you have to hard code values like "top-40." 
Try just using flexbox and center everything